content/de/cloud_computing.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Vorlage für Definition
+status: Feedback erwünscht
+category: Konzept
+---
+
+## Was es ist
+Das Konzept, hinter dem Cloud Computing ist die Bereitstellung von Systemressourcen wie CPU, Netzwerk und Speicher auf Abruf und über das Internet.
+Das Cloud Computing ermöglicht den Zugriff und die Nutzung von Systemressourcen aus beliebigen Standorten.
+Diese Systemressourcen werden von Cloud-Anbietern wie AWS, GCP, Azure, Digitial Ocean, und anderen, aus unterschiedlichen Standorten, an den Endverbraucher vermietet.
+
+## Welches Problem es löst
+Die Skalierung von Systemressourcen stellt für viele Organisationen eine große Herausforderung dar.
+Organisationen müssen entweder fest verplante Systemressourcen bei bestehenden Rechenzentren einkaufen oder ein Rechenzentrum aufbauen und verwalten.
```

Comment:
```suggestion
Organisationen müssen entweder fest verplante Systemressourcen bei bestehenden Rechenzentren einkaufen oder selbst ein Rechenzentrum aufbauen und verwalten.
```

Reply:
@iamNoah1 Vorschlag klingt gut

Reply:
@iamNoah1 angepasst in 81d12e0bbf11431bab9ca67f37314863e7354ca7